python/pyarrow/tests/parquet/test_metadata.py

```diff
@@ -18,12 +18,15 @@
 import datetime
 import decimal
 from collections import OrderedDict
+import os
 
 import numpy as np
 import pytest
 
 import pyarrow as pa
 from pyarrow.tests.parquet.common import _check_roundtrip, make_sample_file
+from pyarrow.filesystem import LocalFileSystem, FileSystem
```

Comment:
Sorry for only noticing this late, but we should import here from `pyarrow.fs` to use the new filesystems in the tests (`pyarrow.filesystem` is deprecated)